aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -59,6 +59,11 @@ Parameters:
     Default: 'true'
     AllowedValues: ['false', 'true']
     Description: 'Enable Sagemaker Notebooks'
+  IAMPartition:
+    Type: String
+    Default: 'aws'
+    AllowedValues: ['aws', 'aws-us-gov']
+    Description: 'IAM Partition (DO NOT CHANGE THIS UNLESS YOU HAVE A VERY SPECIFIC REASON)'
```

Comment:
Can we change the description to `IAM Partition (Select aws-us-gov for AWS GovCloud, otherwise leave as is)`?

Reply:
Sure.  Corrected in new commit, take a look.